src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml.cs

```diff
@@ -101,14 +102,16 @@ private void ContextMenu_OnClose(object sender, RoutedEventArgs e)
 
         private void ContextMenu_OnOpen(object sender, RoutedEventArgs e)
         {
-            // If this context menu just has one member, then if the context menu open, it means all inheritance targets are shown.
-            if (e.OriginalSource is ContextMenu { DataContext: InheritanceMarginViewModel { HasMultipleMembers: false } })
+            if (e.OriginalSource is ContextMenu { DataContext: InheritanceMarginViewModel inheritanceMarginViewModel }
+                && inheritanceMarginViewModel.MenuItemViewModels.Any(vm => vm is TargetMenuItemViewModel))
             {
+                // If the first level of the context menu contains TargetMenuItemViewModel, it means there is only one member on the line,
```

Comment:
nit: This comment doesn't align with the assertion. `.Any` would mean that any of the MenuItemViewModels could be a TargetMenuItemViewModel and get here.

Reply:
I would edit the comment here